lib/cartopy/io/__init__.py

```diff
@@ -100,29 +100,29 @@ class Downloader(object):
 
     Args:
 
-        ``url_template`` - The template of the full URL representing this
-                           resource.
+        * ``url_template``: The template of the full URL representing this
```

Comment:
I think these no longer need to be in code formatting (seems to apply to this entire file.)

Reply:
Good spot.  Will fix this.